kotlin/identify-raster-cell/README.md

```diff
@@ -0,0 +1,39 @@
+# Identify raster cell
+
+Get the cell value of a local raster at the tapped location and display the result in a callout.
+
+![Image of identify raster cell](identify-raster-cell.png)
+
+## Use case
+
+You may want to identify a raster layer to get its exact cell value in the case the approximate value conveyed by its symbology is not sufficient. The information available for the raster cell depends on the type of raster layer being identified. For example, a 3-band satellite or aerial image might provide 8-bit RGB values, whereas a digital elevation model (DEM) would provide floating point z values. By identifying a raster cell of a DEM, you can retrieve the precise elevation of a location.
+
+## How to use the sample
+
+Tap or double tap drag an area of the raster to identify it and see the raster cell attributes information displayed in a callout.
+
+## How it works
+
+1. Create an `DefaultMapViewOnTouchListener` on the `MapView`.
+2. On tap or double tap drag:
+   * Dismiss the `Callout`, if one is showing.
+   * Call `identifyLayerAsync(...)` passing in the raster layer, screen point, tolerance, and maximum number of results per layer.
+   * Add a done loading listener for the result of the identify and then get the `GeoElement` from the layer result.
```

Comment:
It took some work to understand what was happening here, because the elements are immediately filtered into a list of RasterCells. It might be worth adding that to this line here, ie. "get the `GeoElement`s from the layer result and get any `RasterCell`s from them.